app/views/complaints/_tta_report_list.html.erb

```diff
@@ -10,10 +10,5 @@
         <% end %>
       </p>
     </span>
-    <span class="display-none">
```

Comment:
Should there be a replacement for this?

Reply:
no, that design is changing i think to make it easier to link/unlink individual activity reports. (laura and i discussed that last week)